editor/plugins/theme_editor_plugin.cpp

```diff
@@ -2080,7 +2080,7 @@ ThemeItemEditorDialog::ThemeItemEditorDialog(ThemeTypeEditor *p_theme_type_edito
 	List<String> ext;
 	ResourceLoader::get_recognized_extensions_for_type("Theme", &ext);
 	for (const String &E : ext) {
-		import_another_theme_dialog->add_filter(vformat("*.%s; %s", E, TTR("Theme Resource")));
+		import_another_theme_dialog->add_filter(vformat("*.%s", E), TTR("Theme Resource"));
```

Comment:
Same as with audio bus layout. This can be changed to match the style of other filter definitions.